aligner/helper_scripts/filter_scp.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python
```

Comment:
Mentioned elsewhere, but this should be a python function that takes parameters

i.e. with the signature:

def filter_scp_by_utterance(data, utterances)

I don't think the exclude flag is necessary for our purposes, but I could be wrong.